package/MDAnalysis/coordinates/DCD.py

```diff
@@ -172,8 +172,57 @@ def dimensions(self, box):
 
 
 class DCDWriter(base.WriterBase):
-    """Writes to a DCD file
-
+    """Write to a CHARMM/NAMD DCD trajectory file.
+
+    Parameters
```

Comment:
Honestly, I don't remember a discussion, I might have missed that. I don't see anything in the [Style Guide: Docs](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#writing-docstrings).

My reasoning was that with the extended numpy sections it looks better to have everything in the class docs instead of `__init__`, especially as sometimes we might not even have `__init__` for derived classes.

I certainly followed this reasoning in some of the analysis module doc rewrites where class docs contained background and examples and `__init__` contained parameter lists.

Can you point me to where we decided using `__init__` for the class docs? For right now I'd like to leave it as it is.